src/ws/cockpithandlers.c

```diff
@@ -302,9 +302,11 @@ add_page_to_environment (JsonObject *object)
     }
 
   require_host = cockpit_conf_bool ("WebService", "RequireHost", FALSE);
+  gboolean hide_login = cockpit_conf_bool ("WebService", "HideLogin", FALSE);
```

Comment:
Sorry for raising a naming discussion again, but as this introduces config/API: In essence, this enables "client mode" on the login page, right? Which particularly means "only give me an SSH address/host target". So it's still a log in, just not user/password.

So this also conceptually collides with `RemoteHost=`, as you already wrote in the commit message. In https://bugzilla.redhat.com/show_bug.cgi?id=2018741 there was some discussion about fixing that option to also allow the user to entirely disable remote hosts. Do you think we could make this compatible with that approach and have e.g.

    RemoteHost=require|allow|no|only

with "only" meaning that the login page only shows the host field, and nothing else? I'm of course happy to change the values here, but I think it makes sense to unify mutually exclusive config keys into such an enum.

Of course, for this PR we'd only implement the `=only` value.

Also, this needs documentation.

Reply:
I like this suggestion.  I'll look into it.

Reply:
FTR, for both here and PR #16630 -- if this is too entangled, I'd be happy to temporarily use an undocumented [WebService] `_Internal_Client = yes`  flag, and merge this into RequireHost= later.